src/screens/ForgotPassword/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Text, View,Button} from 'react-native';
+import Title from './../../components/Title/index';
+export default ForgotPassword = ({navigation}) => {
+  return (
+    <View
+      style={{
+        flex: 1,
+        justifyContent: 'center',
+      }}>
+      <Title value="Forgot Password"></Title>
+      <Button title="Login" onPress={()=> navigation.navigate('Login')}></Button>
```

Comment:
try to always use a named function instead of an anonymous function. The benefit from that will be like right now we have only a single line that is navigation.navigate('Login') on the press but later on, we need to add more lines to it. so we won't be doing there then also we need to create a named function for that. So, try to create it from the start. Also, you can explore the difference between both.